package.json

```diff
@@ -1954,7 +1966,13 @@
                     "when": "jupyter.hasNativeNotebookOpen"
                 }
             ]
-        }
+        },
+        "debuggers": [
+            {
+                "type": "kernel",
+                "label": "Kernel Debug"
```

Comment:
II think this should be called `Jupyter Python` or `Python Kernel` or similar, as it's specific to python not just any kernel.
This will matter in full debugging, user needs to select the right debug config (based on the label)

Reply:
I changed it to Python Kernel on the cleanup commit